docs/content/docs/test_framework/Mocking.md

```diff
@@ -52,6 +52,27 @@ class MyAwesomeTests {}
 
 This annotation is available in all types of tests but make sense mostly for functional tests.
 
+## Mocking Preferences
+
+All kind of tests are using real implementations of `com.avito.android.util.preferences.Preferences`, not mocked ones.
+But it's unnecessary to mock `Preferences` somehow because all test data is [cleared]({{< ref "/docs/test/TestDataClearing.md" >}}) between tests,
+including databases and `SharedPreferences` files.
+
+All entities that depend on `Preferences` are gathered together in `PreferencesRule`. You can access that entities without going into details of `Preferences`
```

Comment:
i prefer to remove both sentences, they are telling no more than a code snippet below

Reply:
The first sentence tries to explain how test state is cleared between runs. The second one tries to explain that `PreferencesRule` is not about Preferences directly but about something that relies on Preferences. Something more upper-level.

Reply:
Remove first sentences but left second. It seems to me that the second has a meaning for customers.